onnxruntime/python/tools/transformers/models/bart/README.md

```diff
@@ -20,6 +20,16 @@ This folder exports BART model from huggingface to ONNX, specifically on `BartFo
   - [Required Parameters](#required-parameters)
   - [Optional Parameters](#optional-parameters)
 
+### Introduction
+
+We break BART into 3 parts: encoder, decoder, and beam search. As long as you confirm
+that you have the certain part of the model, you are allowed to skip it with parameter.
+
+1. no_encoder (skip converting encoder)
+2. no_decoder (skip converting decoder)
+3, no_chain (skip cobining encoder/decoder with beam seach)
```

Comment:
[misspell] reported by reviewdog 🐶 "seach" is a misspelling of "search"

Reply:
The dog is smart.

Reply:
Done